### PR TITLE
timing tests seperated into their own suite

### PR DESCRIPTION
**Pull request by jassigill2000**

### CODE REVIEW

File: scripts/irods/test/timing_tests.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    import unittest2 as unittest
+else:
+    import unittest
+
+from ..test.command import assert_command
+from ..configuration import IrodsConfig
+from ..controller import IrodsController
+from ..core_file import temporary_core_file, CoreFile
+from .. import test
+from . import settings
```
Comment (trel) [inline scripts/irods/test/timing_tests.py]: Codacy Issue found: [Unused import settings](https://app.codacy.com/manual/trel/irods/pullRequest?prid=5154840)

File: scripts/irods/test/timing_tests.py [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+from .. import lib
+from .. import paths
```
Comment (trel) [inline scripts/irods/test/timing_tests.py]: Codacy Issue found: [Unused import paths](https://app.codacy.com/manual/trel/irods/pullRequest?prid=5154840)

File: scripts/irods/test/timing_tests.py [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+if sys.version_info < (2, 7):
+    import unittest2 as unittest
+else:
+    import unittest
+
+from ..test.command import assert_command
+from ..configuration import IrodsConfig
+from ..controller import IrodsController
+from ..core_file import temporary_core_file, CoreFile
```
Comment (trel) [inline scripts/irods/test/timing_tests.py]: Codacy Issue found: [Unused CoreFile imported from core_file](https://app.codacy.com/manual/trel/irods/pullRequest?prid=5154840)

File: scripts/irods/test/timing_tests.py [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+import ustrings
+
+if sys.version_info < (2, 7):
+    import unittest2 as unittest
+else:
+    import unittest
+
+from ..test.command import assert_command
+from ..configuration import IrodsConfig
+from ..controller import IrodsController
```
Comment (trel) [inline scripts/irods/test/timing_tests.py]: Codacy Issue found: [Unused IrodsController imported from controller](https://app.codacy.com/manual/trel/irods/pullRequest?prid=5154840)

File: scripts/irods/test/timing_tests.py [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+from ..core_file import temporary_core_file, CoreFile
+from .. import test
+from . import settings
+from .. import lib
+from .. import paths
+from .resource_suite import ResourceBase
+from . import session
+from .rule_texts_for_tests import rule_texts
```
Comment (trel) [inline scripts/irods/test/timing_tests.py]: Codacy Issue found: [Unused rule_texts imported from rule_texts_for_tests](https://app.codacy.com/manual/trel/irods/pullRequest?prid=5154840)

File: scripts/irods/test/timing_tests.py [hunk 6]
```diff
@@ -0,0 +1,144 @@
+from __future__ import print_function
+
+import os
+import shutil
+import subprocess
+import sys
+from threading import Timer
+import time
```
Comment (trel) [inline scripts/irods/test/timing_tests.py]: Codacy Issue found: [Unused import time](https://app.codacy.com/manual/trel/irods/pullRequest?prid=5154840)

File: scripts/irods/test/timing_tests.py [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+import ustrings
+
+if sys.version_info < (2, 7):
+    import unittest2 as unittest
+else:
+    import unittest
+
+from ..test.command import assert_command
```
Comment (trel) [inline scripts/irods/test/timing_tests.py]: Codacy Issue found: [Unused assert_command imported from test.command](https://app.codacy.com/manual/trel/irods/pullRequest?prid=5154840)

File: scripts/irods/test/timing_tests.py [hunk 8]
```diff
@@ -0,0 +1,138 @@
+from __future__ import print_function
+
+import os
+import shutil
+import subprocess
+import sys
+from threading import Timer
+import time
```
Comment (trel) [inline scripts/irods/test/timing_tests.py]: Codacy Issue found: [Unused import time](https://app.codacy.com/manual/trel/irods/pullRequest?prid=5154840)

**Comment by trel** [comment]
looks good - please #

**Comment by jassigill2000** [comment]
This can be merged.

**Pull request merged**